languages/go/reference/exercise-concepts/parallel-letter-frequency.md

```diff
@@ -0,0 +1,29 @@
+# Concepts of Parallel Letter Frequency
+
+## Concurrency
+
+The goal of this exercise is to learn to write concurrent code to count the letters inside different texts.
+
+## Goroutines
+
+In order to count concurrent the apppearance of different letters in separate texts it is important to be able to create goroutines with the `go` keyword, one for each text where the letters are counted.
+
+## Channel
+
+In this exercise the student can learn to use a channel to make the main goroutine communicate with the goroutines that count letters.
+
+## Functions
+
+The function that has to be called concurrently to count needs to be created in order to get the total count at the end.
+
+### Anonymous functions
+
+However, since it is a simple function it is not necessary to define an external function but it can be coded inside the `go` statement.
```

Comment:
Indeed, it should be, because a function literal is a closure over the channel variable, so we don't have to pass the channel as an argument as we would using a named function.